docker/docker-compose.yml

```diff
@@ -3,18 +3,14 @@ version: '3.2'
 services:
     oms-core-elixir:
         build:
-            context: ./$PATH_OMS_CORE_ELIXIR/oms-core-elixir/
-            dockerfile: ./Dockerfile.dev
-        volumes:
-            - ./$PATH_OMS_CORE_ELIXIR/../:/usr/src/myapp
+            context: ./$PATH_OMS_CORE_ELIXIR/../
         depends_on:
             - postgres-oms-core-elixir
-        expose:
-            - "4000"
+        ports:
+            - 4000:4000
```

Comment:
I think this is super security risky, I would not expose that directly. I'd rather have it behind the traefik proxy.

Reply:
It was just to have vagrant alert when ready by means of curl (see [here](https://github.com/AEGEE/oms-docker/blob/acdd011b0a1a2b251f92297bbd26198e83cbe4fc/vagrant-post-script/orchestrate_docker.sh))

Reply:
you can still query it through traefik

Reply:
also you will not get a html file from core, it's a json api. If you want I can also implement a status endpoint which checks for DB connectivity

Reply:
I support that actually, we can query it through /services/oms-core-elixir/api

Reply:
Phoenix replies with an HTML saying route not found

I agree with the status endpoint, it can be used by healthchecks

Reply:
also can we make any api be just `/api`? the url is unnecessarily long

Reply:
if there's no html, what's [this](https://github.com/AEGEE/oms-core-elixir/tree/master/frontend)?

Reply:
seems like parts of the old frontend (AngularJS one)